### ISSUE TITLE
"keys.php"

### ISSUE THREAD

**Issue by KirillKo2877** [ahmed-debbech/uno_online#3]
Hey, today I wanted to add this game to my personal website to play with friends, but for some reason "keys.php" file is missing and i can't seem to find it anywhere. Could you please provide code for the file? Thanks.

**Comment by ahmed-debbech** [comment]
Hey, That file contains all the necessary credentials to authenticate to servers and databases. It is a critical file that's why it's never been pushed.

**Comment by KirillKo2877** [comment]
Yeah, that's obvious, though other people usually just edit out the credentials in the file and publish it like that.

**Comment by ahmed-debbech** [comment]
You are correct, but i chose to not commit the file at all.

**Comment by 0P3N50URC3-F0R3V3R** [comment]
So if i get this correctly.. you host something on a git that is totally unusable since you decided it??! Why dont you delete the whole stuff then?! Why is this even here if you dont want to host it by other people...? 

Or at least provide a sample keys.php...

**Comment by 0P3N50URC3-F0R3V3R** [comment]
> Hey, today I wanted to add this game to my personal website to play with friends, but for some reason "keys.php" file is missing and i can't seem to find it anywhere. Could you please provide code for the file? Thanks.

Sample keys.php

```
<?php
$serverIp='127.0.0.1';
$username='root';
$pass='root';
$dbName='uno';
?>
```

Wasn't that hard, since the php errors tell you exactly what it needs.

IDK why dear achmed-debbech wont do it like this.... sample file... if you have problem sharing sensitive information just change it to something else, if it is on github already... OH and use PHP7.x for this.

I have to mention that purely relying on mysql+php is a very roboust and bad idea... it has performance issues... :\ But tha script overall would be nice...

**Comment by ahmed-debbech** [comment]
> > Hey, today I wanted to add this game to my personal website to play with friends, but for some reason "keys.php" file is missing and i can't seem to find it anywhere. Could you please provide code for the file? Thanks.
> 
> Sample keys.php
> 
> ```
> <?php
> $serverIp='127.0.0.1';
> $username='root';
> $pass='root';
> $dbName='uno';
> ?>
> ```
> 
> Wasn't that hard, since the php errors tell you exactly what it needs.
> 
> IDK why dear achmed-debbech wont do it like this.... sample file... if you have problem sharing sensitive information just change it to something else, if it is on github already... OH and use PHP7.x for this.
> 
> I have to mention that purely relying on mysql+php is a very roboust and bad idea... it has performance issues... :\ But tha script overall would be nice...

First of all this thanks for your interest in Uno Online. 

Second of all this whole thing was meant to be a learning experience for me, so back then it wasn't clear to me as a real open source project people getting interest in.

And third, i was just starting out in building websites and applications 5 years ago, now that i am 5 years ahead i gained more experience and thus i am aware of the mysql+php is not a good idea maybe i could have used Redis and websockets.

Anyways i appreciate your time trying to make the project more meaningful.
